src/GregClient/GregClient.csproj

```diff
@@ -33,68 +39,34 @@
-      <HintPath>..\packages\RestSharp.106.12.0\lib\net452\RestSharp.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' OR '$(TargetFramework)' == 'net48'">
     <Reference Include="System.Configuration" />
-    <Reference Include="System.Core" />
     <Reference Include="System.IO.Compression" />
     <Reference Include="System.IO.Compression.FileSystem" />
     <Reference Include="System.Web" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AuthProviders\BasicProvider.cs" />
-    <Compile Include="AuthProviders\IAuthProvider.cs" />
-    <Compile Include="AuthProviders\LoginState.cs" />
-    <Compile Include="Converters\DependencyConverter.cs" />
-    <Compile Include="IGregClient.cs" />
-    <Compile Include="PackageManagerClient.cs" />
-    <Compile Include="Properties\globalVersionInfo.cs" />
-    <Compile Include="Requests\BanPackage.cs" />
-    <Compile Include="Requests\Deprecate.cs" />
-    <Compile Include="Requests\Downvote.cs" />
-    <Compile Include="Requests\GetMaintainers.cs" />
-    <Compile Include="Requests\HeaderVersionDownload.cs" />
-    <Compile Include="Requests\Hosts.cs" />
-    <Compile Include="Requests\PackageManagerRequest.cs" />
-    <Compile Include="Requests\PackageReferenceRequest.cs" />
-    <Compile Include="Requests\PackageUploadRequestBody.cs" />
-    <Compile Include="Requests\PackageVersionUploadRequestBody.cs" />
-    <Compile Include="Requests\Search.cs" />
-    <Compile Include="Requests\JSONRequest.cs" />
-    <Compile Include="Requests\PackageDownload.cs" />
-    <Compile Include="Requests\TermsOfUse.cs" />
-    <Compile Include="Requests\Undeprecate.cs" />
-    <Compile Include="Requests\Upvote.cs" />
-    <Compile Include="Utility\FileUtilities.cs" />
-    <Compile Include="Requests\HeaderCollectionDownload.cs" />
-    <Compile Include="Requests\HeaderDownload.cs" />
-    <Compile Include="Requests\RequestBody.cs" />
-    <Compile Include="Requests\Request.cs" />
-    <Compile Include="Requests\PackageUpload.cs" />
-    <Compile Include="GregClient.cs" />
-    <Compile Include="Requests\PackageVersionUpload.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Requests\ValidateAuth.cs" />
-    <Compile Include="Responses\Responses.cs" />
-    <Compile Include="Utility\PackageUtilities.cs" />
-    <Compile Include="Utility\Utilities.cs" />
+  <ItemGroup Label="Common package dependencies">
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="RestSharp" Version="106.12.0" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
-    <None Include="packages.config" />
+    <None Include="..\..\nuspec\images\logo_square_32x32.png">
+      <Pack>True</Pack>
+      <PackagePath>\</PackagePath>
+    </None>
+  </ItemGroup>
+  <ItemGroup Label="Compile time reference assemblies">
```

Comment:
I think this is unnecessary and you can just reference the meta package, and it will pull the others depending on current target:

https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/

also in the docs, I see no use of the `IncludeAssets` element...?

https://github.com/Microsoft/dotnet/tree/master/releases/reference-assemblies#sdk-style-project

Reply:
Oh I see. I missed that
Found the explanation here https://andrewlock.net/using-reference-assemblies-to-build-net-framework-libararies-on-linux-without-mono/
Looks like the minimum requirement will be dotnet sdk vers x.y to be installed 
Reagarding the IncludeAssets....that is not needed since it is set to `all` by default